core/src/jsMain/kotlin/Bluetooth.kt

```diff
@@ -0,0 +1,26 @@
+package com.juul.kable
+
+import com.juul.kable.external.Bluetooth
+import kotlinx.coroutines.CoroutineScope
+import kotlin.js.Promise
+
+private val bluetooth: Bluetooth
+    get() = js("window.navigator.bluetooth") ?: error("Bluetooth unavailable")
```

Comment:
Oh. Good point. I don't believe this will work for Node.js.
Looks like Noble library does it via some kind of Node.js bindings mechanism.

https://github.com/noble/noble/blob/c4cd18a7a429bb832f6c4ca793be3faf9af884e9/lib/webbluetooth/bindings.js#L28-L45

...something to look into in the future. Tracking via #25.